packages/runtime/client/rest/src/payout/endpoints/singleton.ts

```diff
@@ -0,0 +1,36 @@
+
+import * as E from 'fp-ts/lib/Either.js'
+import * as TE from 'fp-ts/lib/TaskEither.js'
+import { pipe } from 'fp-ts/lib/function.js';
+import * as t from "io-ts/lib/index.js";
+
+import { AxiosInstance } from 'axios';
+
+import {formatValidationErrors} from 'jsonbigint-io-ts-reporters'
+
+import * as HTTP from '@marlowe.io/adapter/http';
+import { DecodingError } from '@marlowe.io/adapter/codec';
+
+import { PayoutId, unPayoutId } from "@marlowe.io/runtime-core";
+import { PayoutDetails } from '../details.js';
+
+export type GET = ( payoutId: PayoutId) => TE.TaskEither<Error | DecodingError, PayoutDetails>
+
+type GETPayload = t.TypeOf<typeof GETPayload>
+const GETPayload = t.type({ links: t.type ({}), resource: PayoutDetails})
+
+export const getViaAxios:(axiosInstance: AxiosInstance) => GET
```

Comment:
Not sure what this is supposed to do but the name is a little bit of a code smell.

Should this be called `getPayoutDetails`? what has Axios have to do with it?